src/main/resources/application.properties

```diff
@@ -21,8 +21,14 @@ spring.datasource.hikari.maximum-pool-size=${DATA_STORE_DB_MAX_POOL_SIZE:16}
 spring.jpa.properties.hibernate.temp.use_jdbc_metadata_defaults = false
 pagination.page.size=25
 
+# Access Management Postgres DB"
+am.datasource.url=jdbc:postgresql://${AM_DB_HOST:localhost}:${AM_DB_PORT:5600}/${AM_DB_NAME:am}
+am.datasource.username=${AM_DB_USERNAME:amuser}
+am.datasource.password=${AM_DB_PASSWORD:ampass}
+
```

Comment:
I would have expected the am db config to have similar set of properties as the ccd db config:
`
spring.jpa.show-sql=false
spring.main.allow-bean-definition-overriding=true
spring.datasource.url=jdbc:postgresql://${DATA_STORE_DB_HOST:localhost}:${DATA_STORE_DB_PORT:5452}/${DATA_STORE_DB_NAME:ccd_data}${DATA_STORE_DB_OPTIONS:?stringtype=unspecified}
spring.datasource.username=${DATA_STORE_DB_USERNAME:ccd}
spring.datasource.password=${DATA_STORE_DB_PASSWORD:ccd}
#spring.datasource.tomcat.max-active=30
spring.datasource.type=com.zaxxer.hikari.HikariDataSource
spring.jpa.hibernate.ddl-auto=none
spring.datasource.hikari.connection-timeout=${DATA_STORE_DB_CONNECTION_TIMEOUT:40000}
spring.datasource.hikari.idle-timeout=${DATA_STORE_DB_IDLE_TIMEOUT:300000}
spring.datasource.hikari.minimum-idle=${DATA_STORE_DB_MIN_IDLE:8}
spring.datasource.hikari.maximum-pool-size=${DATA_STORE_DB_MAX_POOL_SIZE:16}
# Disable feature detection to avoid the java.sql.SQLFeatureNotSupportedException
# Method org.postgresql.jdbc.PgConnection.createClob() is not yet implemented.
spring.jpa.properties.hibernate.temp.use_jdbc_metadata_defaults = false
`

Reply:
Also I cannot see any changes in `docker-compose.yml`.

In a this PR that also sets up 2 data sources (https://github.com/hmcts/ccd-definition-store-api/pull/323) for def store we have changes in `application.yml`:
`accessmanagement:
  enabled: ${ACCESS_MANAGEMENT_EXPORT_ENABLED:true}
spring:
  datasource:
    driver-class-name: org.postgresql.Driver
    url: jdbc:postgresql://${AM_DB_HOST:localhost}:${AM_DB_PORT:5433}/${AM_DB_NAME:am}${AM_DB_PARAMS:}
    username: ${AM_DB_USERNAME:amuser}
    password: ${AM_DB_PASSWORD:ampass}
    properties:
      charSet: UTF-8`

Reply:
Updated the Application.yml to use the similar set as CCD DB.

# Access Management Postgres DB"
am.datasource.jdbc-url=jdbc:postgresql://${AM_DB_HOST:localhost}:${AM_DB_PORT:5600}/${AM_DB_NAME:am}
am.datasource.username=${AM_DB_USERNAME:amuser}
am.datasource.password=${AM_DB_PASSWORD:ampass}
am.datasource.type=com.zaxxer.hikari.HikariDataSource
am.datasource.hikari.connection-timeout=${AM_DB_CONNECTION_TIMEOUT:40000}
am.datasource.hikari.idle-timeout=${AM_DB_IDLE_TIMEOUT:300000}
am.datasource.hikari.minimum-idle=${AM_DB_MIN_IDLE:8}
am.datasource.hikari.maximum-pool-size=${AM_DB_MAX_POOL_SIZE:10}
am.liquibase.enabled=false

Working on the changes in docker-compose.yml.